ergoCubSN000/hardware/battery/battery.xml

```diff
@@ -4,6 +4,7 @@
 <device xmlns:xi="http://www.w3.org/2001/XInclude" name="canbattery" type="embObjBattery">
 
     <xi:include href="../../general.xml" />
+    <xi:include href="../electronics/left_arm-eb2-j0_1-eln.xml" />
```

Comment:
Why it is called `left_arm...` ?

Reply:
Talking with @AntonioConsilvio, he said me that the net board is the ems that controls the left arm (i.e. `eb2`)